analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Tests.Diagnostics
+{
+    class Tests
+    {
+
+        private int Length { get; }
+
+        void Test(List<string> foo)
+        {
+            bool result = foo.Contains("foo");
```

Comment:
Add a comment to explain that this is needed to trigger the rule via `ShouldExecute`.